src/Aspire.Dashboard/Configuration/DashboardOptions.cs

```diff
@@ -15,6 +15,7 @@ public sealed class DashboardOptions
     public FrontendOptions Frontend { get; set; } = new FrontendOptions();
     public ResourceServiceClientOptions ResourceServiceClient { get; set; } = new ResourceServiceClientOptions();
     public TelemetryLimitOptions TelemetryLimits { get; set; } = new TelemetryLimitOptions();
+    public OpenIdConnectOptions OpenIdConnect { get; set; } = new OpenIdConnectOptions();
```

Comment:
@JamesNK @drewnoakes I'm open to suggestions on how these options should be configured. I didn't see (though definitely could have missed) a way to handle it within the OpenIDConnect infrastructure. I know there's further work on configuring policies, etc.

Reply:
Moved it.